source/developer-guides/configuration.rst

```diff
@@ -79,9 +79,11 @@ Platform ID
 
 |SPN| uses platform ID to select the associated configuration data. The platform ID can be specified at build time or dynamically detected from GPIO pins at runtime. At the beginning of Stage 1B (``GetBoardIdFromGpioPins()``), |SPN| attempts to load GPIO platform ID by tag ``CDATA_PID_GPIO_TAG``. If the tag is found, the actual platform ID value is read from the GPIO pins. Otherwise, |SPN| uses static platform ID.
 
-|SPN| supports up to 32 platform IDs with 16 allocated for static IDs and 16 for dynamic IDs.
+|SPN| supports up to 32 platform IDs with 16 allocated for static IDs and 16 for dynamic IDs. Note that Platform ID **0** is reserved and cannot be used for real board. So technically, SBL can support upto 31 boards.

```

Comment:
we don't have to differentiate static / dynamic platform ID.  It is up to platform to map them accordingly.
Also Platform ID 0 is served to carry the default CFGDATA values defined in the CfgDataDef.dsc file. So it cannot be used for a real board.